### ISSUE TITLE
getAttachmentContentAsync "the operation is not supported"

### ISSUE THREAD

**Issue by GarryLowther** [OfficeDev/office-js#718]
The call to getAttachmentContentAsync works perfectly in Outlook 365 Web however it fails in Windows Installed Outlook 365 ProPlus Version 1902 (Build 11328.20392 Click-to-Run)

## Expected Behavior
I expect this to work THE SAME as in Outlook 365 Web

## Current Behavior
Throws a "Host Error: the operation is not supported

## Steps to Reproduce, or Live Example
```
var options = { asyncContext: { type: attachment.attachmentType } };
item.getAttachmentContentAsync(attachment.id, options, function (result) 
{
	if (result.status == Office.AsyncResultStatus.Succeeded) 
	{
		//console.log("Call returned success!");
		var AttachmentContent = result.value; // Get the attachment content
		if (AttachmentContent.format == Office.MailboxEnums.AttachmentContentFormat.Base64) 
		{
			// handle file attachment
			//console.log(attachment.name + " Base64 String: " + AttachmentContent.content);
			TriSysApex.Toasters.Error("Downloaded " + attachment.name);
			fnCallback(AttachmentContent.content);
		}
		else if (result.format == Office.MailboxEnums.AttachmentContentFormat.Eml) {
			// handle item attachment
		}
		else {
			// handle cloud attachment  
		}
	}
	else 
	{
		var err = result.error;
		TriSysApex.Toasters.Error("Download failed: " + err.name + ": " + err.message);
	}
});
```

## Context
This is a show-stopper as the whole point of having OfficeJS is to write code once. So this code allows customers to download attachments only in Office365 Web, but not in installed Outlook on Windows.

## Your Environment
Windows 10 64-bit, Office 365, Edge (Chromium)

**Comment by AlexJerabek** [comment]
Hi @exextoc, can you please investigate?

**Comment by GarryLowther** [comment]
Your documentation at https://docs.microsoft.com/en-us/javascript/api/outlook/office.attachmentdetails?view=outlook-js-1.7 only shows how to list attachments, not to download them. 
You do have separate documentation at https://docs.microsoft.com/en-us/javascript/api/outlook/office.item?view=outlook-js-preview#getattachmentcontentasync-attachmentid--options--callback-
There is also a similar bug related to getAttachmentContentAsync at https://github.com/OfficeDev/office-js/issues/644

**Comment by AlexJerabek** [comment]
Thanks for reporting this @TriSysBusinessSoftware. I'll also tag @ElizabethSamuel-MSFT to look into the documentation.

**Comment by ElizabethSamuel-MSFT** [comment]
Per the [preview page](https://docs.microsoft.com/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview?view=outlook-js-preview#attachments), this API should be supported in web and Windows.

@exextoc Let me know if I need to update the docs. Thx.

**Comment by exextoc** [comment]
The build number you mentioned happens to be a semi-annual channel build number. For those channels, this API is only supported for builds greater than **16.0.11716.1000**. For the monthly channel however, builds greater than **16.0.10901.1000** are supported. Also note that this API is in preview still and may be prone to bugs until it's released as a part of a requirement set.

**Comment by GarryLowther** [comment]
Those version numbers (16.0....) do not correspond to those documented here: https://docs.microsoft.com/en-us/officeupdates/update-history-office365-proplus-by-date
Is there a 'mapping' between 16 versions and Office 365 versions? If so, how can I view these?

**Comment by GarryLowther** [comment]
![image](https://user-images.githubusercontent.com/5684253/64695342-cca29980-d493-11e9-8e3b-bcaabeee1abf.png)
I can confirm that downloading attachments works in this version of Outlook.

**Comment by SanketDG** [comment]
We are also facing this issue with Outlook 365 Windows Desktop Client, we do have the right Outlook version as mentioned below:

![MicrosoftTeams-image](https://user-images.githubusercontent.com/8980971/96454135-0f0edc00-1239-11eb-8ee9-834c11560042.png)

Is there any update on this?

**Comment by exextoc** [comment]
@SanketDG The OP confirmed that the original issue was resolved in Version 1908 Build 11929.20300. Could you open a new issue with a code sample for the API that's failing?

[getAttachmentContextAsync](https://docs.microsoft.com/en-us/javascript/api/outlook/office.messageread?view=outlook-js-1.8&preserve-view=true#getattachmentcontentasync-attachmentid--options--callback-) was released with [Requirement Set 1.8](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/objectmodel/requirement-set-1.8/outlook-requirement-set-1.8) and should be available in any Outlook version supporting that requirement set.
